documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -4210,6 +4209,275 @@ After the connector's commit SCN advances beyond the DDL change, to prevent unpa
+=== Obtaining the Oracle JDBC driver for OpenLogReplicator
+
+The {prodname} Oracle connector requires the Oracle JDBC driver to connect to Oracle databases.
+Please refer to the link:#obtaining-the-oracle-jdbc-driver[Obtaining the Oracle JDBC driver] as OpenLogReplicator has the exact same requirements as Oracle LogMiner.
+
+[[oracle-openlogreplicator-configuration]]
+=== OpenLogReplicator configuration
+
+OpenLogReplicator is a third-party standalone process that is responsible for reading the Oracle redo and archive logs.
+Unlike other {prodname} Oracle adapters, this means that there are two configurable components that must be configured independently of one another.
+
+OpenLogReplicator is configured by using a JSON-based configuration in a file called `scripts/OpenLogReplicator.json`.
+The format of this JSON configuration can be found https://github.com/bersler/OpenLogReplicator/blob/master/documentation/reference-manual/reference-manual.adoc#openlogreplicator-json-file-format[here].
+
+.OpenLogReplicator configuration
+[source,json,indent=0]
+----
+{
+  "version": "1.2.1",
+  "source": [{
+    "alias": "SOURCE",
+    "name": "ORACLE", <.>
+    "reader": {
+      "type": "online",
+      "path-mapping": ["/opt/olr/recovery_area", "/opt/olr/recovery_area/ORCLCDB/archivelog"], <.>
+      "user": "c##dbzuser", <.>
+      "password": "dbz", <.>
+      "server": "//<ip>:<port>/ORCLPDB1" <.>
+    },
+    "format": { <.>
+      "type": "json",
+      "column": 2,
+      "db": 3,
+      "interval-dts": 9,
+      "interval-ytm": 4,
+      "message": 2,
+      "rid": 1,
+      "schema": 7,
+      "scn-all": 1,
+      "timestamp-all": 1
+    }
+  }],
+  "target": [{
+    "alias": "DEBEZIUM",
+    "source": "SOURCE",
+    "writer": {
+      "type": "network", <.>
+      "uri": "<host>:<port>" <.>
+    }
+  }]
+}
+----
+<.> This should match the `openlogreplicator.source` connector configuration.
+<.> List of file path pairs `[before1,after1,befor2,after2,...]` where if a log file path matches one of the `beforeX` prefixes, the prefix is replaced with the `afterX` path.  This is useful when OpenLogReplicator is running on a different host than the source database and the path to the redo and archive logs different between the database and the OpenLogReplicator process.
+<.> This should match the `database.user` connector property.
+<.> This should match the `database.password` connector property.
+<.> This should point to the database host, port, and Oracle SID.
+<.> This specifies the payload format ingested by {prodname} Oracle connector. Use these values as specified, as these are the only format options that we require beyond the defaults.
+<.> This must specify `network`, as {prodname} Oracle connector communicates with OpenLogReplicator via a network connection.
+<.> This specifies the bind host and port that OpenLogReplicator listens for connections on.  This should be accessible by {prodname} Oracle connector.
```

Comment:
```suggestion
<1> This should match the `openlogreplicator.source` connector configuration.
<2> List of file path pairs `[before1,after1,befor2,after2,...]` where if a log file path matches one of the `beforeX` prefixes, the prefix is replaced with the `afterX` path.  This is useful when OpenLogReplicator is running on a different host than the source database and the path to the redo and archive logs different between the database and the OpenLogReplicator process.
<3> This should match the `database.user` connector property.
<4> This should match the `database.password` connector property.
<5> This should point to the database host, port, and Oracle SID.
<6> This specifies the payload format ingested by {prodname} Oracle connector. Use these values as specified, as these are the only format options that we require beyond the defaults.
<7> This must specify `network`, as {prodname} Oracle connector communicates with OpenLogReplicator via a network connection.
<8> This specifies the bind host and port that OpenLogReplicator listens for connections on.  This should be accessible by {prodname} Oracle connector.
```

Although the period characters work in the rendered doc, it's best to explicitly identify each callout with a numeral, so that you can easily collate them in the asciidoc source.